src/components/post/TagSelector.styled.ts

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import { motion } from 'framer-motion';
+import { colors } from '@styles/theme/colors';
+
+export const CategorySelector = styled.div<{ $showTags: boolean; $category?: string }>`
+    display: flex;
+    gap: 0.8rem;
+    overflow-x: auto;
+    margin-bottom: ${({ $showTags, $category }) => $showTags && $category ? '1.2rem' : '3.2rem'};
+    
+    /* 스크롤바 숨기기 */
+    -ms-overflow-style: none;
+    scrollbar-width: none;
+    &::-webkit-scrollbar {
+        display: none;
+    }
+`;
+
+export const CategoryChip = styled.div<{ $active?: boolean }>`
```

Comment:
```suggestion
export const CategoryChip = styled.button<{ $active?: boolean }>`
```

`<button>` 태그를 사용하는게 혹시나 의미론적이나, 웹 환경에서 사용할 경우에 Tab으로 이동 가능하면서도 Enter 기능을 사용할 수 있어서 좋을 것 같아요!